test/server/lib/DocApi.ts

```diff
@@ -3969,34 +3967,44 @@ function testDocApi() {
         }
       });
 
-      it("delivers expected payloads from combinations of changes, with retrying and batching",
-        async function () {
-        // Create a test document.
-        const ws1 = (await userApi.getOrgWorkspaces('current'))[0].id;
-        const docId = await userApi.newDoc({name: 'testdoc'}, ws1);
-        const doc = userApi.getDocAPI(docId);
-
-        // For some reason B is turned into Numeric even when given bools
+      async function createWebhooks({docId, tableId, eventTypesSet, isReadyColumn, enabled}:
+        {docId: string, tableId: string, eventTypesSet: string[][], isReadyColumn: string, enabled?: boolean}
+      ) {
+        const subscribeResponses = [];
+        const webhookIds: Record<string, string> = {};
+        // Ensure the isReady column is a Boolean
         await axios.post(`${serverUrl}/api/docs/${docId}/apply`, [
-          ['ModifyColumn', 'Table1', 'B', {type: 'Bool'}],
+          ['ModifyColumn', tableId, isReadyColumn, {type: 'Bool'}],
         ], chimpy);
 
-        // Make a webhook for every combination of event types
-        const subscribeResponses = [];
-        const webhookIds: Record<string, string> = {};
-        for (const eventTypes of [
-          ["add"],
-          ["update"],
-          ["add", "update"],
-        ]) {
+        for (const eventTypes of eventTypesSet) {
           const {data, status} = await axios.post(
-            `${serverUrl}/api/docs/${docId}/tables/Table1/_subscribe`,
-            {eventTypes, url: `${serving.url}/${eventTypes}`, isReadyColumn: "B"}, chimpy
```

Comment:
Happy new year @alexmojaki! :tada: 

I could manage to introduce some tests as requested… However, it takes advantage of this deprecated endpoint API.

Is it OK? 
Or may I convert this piece of code to rather use `POST /webhooks`? (if it's relevant for this endpoint to be called in the `describe('table endpoints'` section)

Reply:
I think what you've done looks fine.